docs/motivation.md

```diff
@@ -277,6 +277,26 @@ assumes the existence of a coalescing queue on input. The client SHOULD NOT
 assume that transactions for different paths are processed in the order in
 which they are received.
 
+### Persistence of Configuration
+
+Like other control-plane protocols, the "dynamic" configuration of a device
```

Comment:
The defniition of dynamic conflicts with dynamic in [bootz](https://github.com/openconfig/bootz/blob/a50e5d4599e34ef3fcfd546ad634a2f48ba54dc2/README.md?plain=1#L39) and [gnmi union_replace](https://github.com/openconfig/reference/blob/master/rpc/gnmi/gnmi-union_replace.md#212-dynamic-configuration).  Could we call this ephemeral or volatile config?

Reply:
I did suspect this would end up being overloaded.

Folks seemed to dislike "volatile" -- but it'd be my preference over "ephemeral". Ephemeral is defined, in rfc7921, thus:

>   Ephemeral state on a router is the state
>   that does not survive the reboot of a routing device or the reboot of
>   the software handling the I2RS software on a routing device. 

Now, if we say that this state should survive control-plane switchover events, then does this mean that it survives "the reboot of the software handling the i2rs software", and what does i2rs mean in this context.

So maybe I prefer volatile here, since it has less connotations in the industry. WDYT?

Reply:
For what it's worth, internally we've been referring to this as "high-velocity input". As you note, "dynamic" is overloaded, and "config" has various other connotations for us (some baked-in assumptions about persistence, being part of "show running-config", etc).

Although admittedly I'm still somewhat partial to yolo-config :)

Reply:
`yolo` is clearly the best choice, but maybe we go with `high-velocity` and just abbreviate this to `hv`? 

This would give us something like `/interfaces/interface/hv/oper-status` for example.

Reply:
On second thoughts, this is a bit cryptic -- so maybe just `volatile`.

We should put together an updated version of `draft-openconfig-netmod-opstate` that has a clear definition of the types of state that exist within OpenConfig at this point in time -- including those that are defined in the `bootz` document.

Reply:
+1 to `volatile`.  It doesn't collide with other commonly used terms and does imply semantics of 'temporary and changes often, quickly, etc.'.  :)